src/www/index.html

```diff
@@ -183,7 +183,7 @@
         </div>
         <hr style="margin-bottom: 0; margin-top: 10px;">
         <div class="custom-menu-item">
-          <a class="custom-icon" href="https://eventyay.com/" target="_blank"><img src="./icons/eventyay.png">
+          <a class="custom-icon" href="https://eventyay.com/" target="_blank"><img src="./favicon.ico">
```

Comment:
@0x48piraj Please use eventyay.png only, instead of changing logo resize it to proper dimensions.

Reply:
@geekyd It's outdated icon.
It been adapted in many projects.